src/pyclaw/limiters/tvd.py

```diff
@@ -22,51 +22,51 @@
 6. Frommm - :math:`1/2 (1 + r)`
 7. Albada 2 - :math:`(r^2 + r) / (1 + r^2)`
 8. Albada 3 - :math:`1/2 (1+r) (1 - (|1-r|^3) / (1+|r|^3))`
```

Comment:
Same thing for this file, I would leave it unchanged and leave the change to another PR.

Reply:
So all these touched files now I realize were accessed by me and I meddled with them, but changed it back to its original state, but it is showing all these as commits because I touched them.